docs/bokeh/switcher.json

```diff
@@ -1,10 +1,15 @@
 [
   {
-    "name": "3.6.0 (latest)",
-    "version": "3.6.0",
+    "name": "3.7.0 (latest)",
+    "version": "3.7.0",
     "url": "https://docs.bokeh.org/en/latest/",
     "preferred": true
   },
+  {
+    "name": "3.6.2",
+    "version": "3.6.2",
+    "url": "https://docs.bokeh.org/en/3.6.2/"
```

Comment:
Sorry for the late response, but the latest point release was `3.6.3`.
```
    "name": "3.6.3",
    "version": "3.6.3",
    "url": "https://docs.bokeh.org/en/3.6.3/"
```
I think this could be updated.